volumina/volumeEditorWidget.py

```diff
@@ -245,6 +245,12 @@ def onShapeChanged():
                 cropMin = (self.editor.posModel.time,self.editor.cropModel._crop_extents[0][0],self.editor.cropModel._crop_extents[1][0],self.editor.cropModel._crop_extents[2][0],0)
                 self.quadViewStatusBar.updateShape5Dcropped(cropMin,self.editor.posModel.shape5D)
 
+            timeDim = self.editor.posModel.shape5D[0]
```

Comment:
Why is this here and not in the _setupVolumeExtent()? Is it because you want to allow for crops with no time to be selected from volumes with time?